src/templates/drizzle/logo.hbs

```diff
@@ -1,6 +1,42 @@
-    </g>
-</svg>
+<div class="sds-logo">
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Layer_1" viewBox="0 0 542.4 154.8">
+        <title>Spark Design System</title>
+        <style>
```

Comment:
Can you have it output the css as attributes instead so the classes don't potentially conflict with other items on the page.

Reply:
I would actually not put this in this file even. Should be in src/assets/drizzle/styles somewhere. That's how I have been keeping Spark styles separate from Drizzle, just adding Drizzle stuff to the CSS that came with Drizzle. FYI, Drizzle auto-namespaces classes you make in there, so if you make a class called `.Logo` it will automatically make it `.drizzle-Logo`

Reply:
ahh ok, how about i move it to a file, then @afebbraro you can fix up the svg later?